setup.py

```diff
@@ -47,15 +47,14 @@
         'numpy<1.20; python_version >= "3.7.0"',
         "protobuf==3.*",
         # 'scipy',
-        "yaspin",
-        "deprecation",
-        "allennlp==0.9.0",
-        "tokenizers",
-        "scikit-learn==0.22.1",
-        "overrides==3.1.0",
-        "boto3==1.15.18",
-        "unittest2",
-        "playsound",
```

Comment:
Why remove `playsound` here when it seems like you've been adding it in everywhere else?